dependencies.gradle

```diff
@@ -1,10 +1,9 @@
 // Add your dependencies here
 
 dependencies {
-    compile ("com.github.GTNewHorizons:Avaritia:1.31:dev")
-	compile ("com.github.GTNewHorizons:Universal-Singularities:8.6.5:dev")
-    compile ("com.github.GTNewHorizons:NotEnoughItems:2.3.12-GTNH:dev")
-	compile ("com.github.GTNewHorizons:CraftTweaker:3.2.9:dev")
-	compile ("com.github.GTNewHorizons:WanionLib:1.8.3:dev")
-
+    implementation("com.github.GTNewHorizons:NotEnoughItems:2.3.28-GTNH:dev")
+    implementation("com.github.GTNewHorizons:Avaritia:1.31:dev")
```

Comment:
implementation is in the runtime classpath of both this project and all projects that depend on it. It's there to speed up compilation of dependants because it doesn't need to be resolved at compile time of projects that depend on this one.

Reply:
ah, we are good then.